src/OctoshiftCLI.Tests/Octoshift/Services/SecretScanningAlertServiceTests.cs

```diff
@@ -740,7 +754,7 @@ public async Task Migrate_Matching_Alerts_With_Different_Resolutions()
             100,
             SecretScanningAlert.AlertStateResolved,
             SecretScanningAlert.ResolutionRevoked,
-            null)
+            "[@] ")
```

Comment:
Since this is a situation which should never happen it'd be better to add the resolver name to the target secret's payload.

Reply:
I've improved the test to make it a bit more clear as I wasn't sure if you're comment was towards the test or the overall unexpected case of source and target alert having different resolution reasons.

If it's the later, agree that it shouldn't happen, but for consistency I suggest we keep the original alerts resolution reason, comment and username as latest update on the target "migrated" alert.

Reply:
Yeah my comment was for the later, but I am 👍 if you think it's better to keep it's better as is. Probably I should've marked this as a nitpick.